en/sensor/pmw3901.md

```diff
@@ -66,7 +65,8 @@ Thone<br>![PMW3901 Thoneflow Notch](../../assets/hardware/sensors/pmw3901/thonef
 <span id="configuration"></span>
 ## PX4 Configuration
 
-The PX4 configuration that is common to all boards:
+The PX4 configuration that is common to all PMW3901-based boards:
+- [SENS_EN_PMW3901](../advanced_config/parameter_reference.md#SENS_EN_PMW3901) must be set `1` in order to enable the sensor driver.
```

Comment:
@DanielePettenuzzo @bkueng Would this parameter be needed for the UAVCAN variant of the sensor?

Reply:
No, not for UAVCAN